docs/core/diagnostics/runtime-events.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Runtime Events"
+description: Review events emitted by CoreCLR runtime.
+ms.date: "09/14/2020"
+ms.topic: reference
+helpviewer_keywords: 
+  - "runtime events [.NET Core]"
+  - "ETW, EventPipe runtime events (CoreCLR)"
+---
+# .NET Core runtime events
+
+For runtime events in .NET Framework, please refer to the [CLR ETW Events document](../../framework/performance/clr-etw-events.md).
+
+## In This Section
+
+ [Contention Events](runtime-contention-events.md)
+ These events collect information about monitor lock contentions.
```

Comment:
These all need to have a `\` marker at the end to create a new line. Otherwise they run as a single sentence.

![image](https://user-images.githubusercontent.com/67293991/94855114-dff80e00-03e2-11eb-933c-f99927f12893.png)

```suggestion
 [Contention Events](runtime-contention-events.md)\
 These events collect information about monitor lock contentions.
```